src/planning/planning/utils/path_3d_rotation.py

```diff
@@ -174,5 +316,7 @@ def animate(i):
     fig, animate, init_func=init, frames=len(t_fine), interval=20, blit=True
 )
 
+anim.save('animation.mp4', writer='ffmpeg')
```

Comment:
Can we save this to out/animation.mp4 since out/ is gitignored

Reply:
Sounds good